web/admin/application/configs/klear/BillableCallHistoricsList.yaml

```diff
@@ -0,0 +1,145 @@
+#include conf.d/mapperList.yaml
+#include conf.d/actions.yaml
+#include conf.d/documentationLink.yaml
+
+production:
+  main:
+    module: klearMatrix
+    defaultScreen: billableCallHistoricsList_screen
+
+  screens: &billableCallHistorics_screensLink
+    billableCallHistoricsList_screen: &billableCallHistoricsList_screenLink
+      controller: list
+      csv: false
+      order:
+        field:
+          - BillableCallHistoric.startTime
+        type: desc
+      pagination:
+        items: 25
+      <<: *BillableCallHistorics
+      class: ui-silk-application-view-list
+      title: _("List of %s %2s", ngettext('External call', 'External calls', 0), "[format| (%parent%)]")
```

Comment:
Is this label correct? or it should contain Historic word? _(multiple times in file)_